api-controllers/ServiceBrokerApiController.js

```diff
@@ -364,6 +355,16 @@ class ServiceBrokerApiController extends FabrikBaseController {
       .catch(NotFound, gone);
   }
 
+  static getDashboardUrl(plan, instanceId) {
```

Comment:
move this static method to utils or make it instance method. We generally dont have any static method in any of the controllers. Also this could be made an instance method. What is the reason to make this static?

Reply:
Kept int his class as might not be used by anyone else. Also is static as called from an inner function **function done() {}** which doesn't allow **this** keyword to use. Could have made inner method. But used in two places.

Reply:
I dont want to insist, but ideally such methods go into utils. Anyways leave it upto your discretion on this.